sharktank/sharktank/models/llama/llama.py

```diff
@@ -46,6 +47,9 @@ class LlamaModelConfig:
     # Dtype to use for attention.
     attention_dtype: torch.dtype = torch.float16
 
+    # Indicates if running with HuggingFace implementation.
```

Comment:
Maybe add a little comment on how HF is different, also maybe call the flag `use_hf`

Reply:
I agree this makes more sense. Changed